server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -799,6 +800,86 @@ public Consumer verifyAndLookupConsumerWithEntitlements(String consumerUuid) {
         return listByCriteria(crit, pageRequest);
     }
 
+    public int countConsumers(String ownerKey,
+        Set<String> typeLabels, List<String> skus, List<String> subscriptionIds, List<String> contracts) {
```

Comment:
Try not to use specific collections here unless it's absolutely required by the implementation.